IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -384,7 +384,7 @@ - (void)decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction
         //Open secure web links with target=new window in default browser or non-web links with URL schemes that can be opened by the application
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->